features/comments_and_kudos/inbox.feature

```diff
@@ -49,6 +49,25 @@ Feature: Get messages in the inbox
       And I press "Filter"
     Then I should not see "guest on Down for the Count"
 
+  Scenario: I can bulk edit comments in my inbox by clicking 'Select'
+    Given I am logged in as "boxer" with password "10987tko"
```

Comment:
You can use `Given I am logged in as "boxer"` since we don't care what the passwords are in this test. You don't have to update the existing tests though.

Reply:
Resolved in ae6fb4b98